devel/ci/Dockerfile-rawhide

```diff
@@ -8,48 +8,60 @@ RUN dnf install --disablerepo rawhide-modular -y \
     findutils \
     git \
     liberation-mono-fonts \
+    python2-yaml \
```

Comment:
I don't think Bodhi directly uses this library.

Reply:
Correct, pyyaml is still listed in the dependencies in `setup.py` though. Should it be removed there?

Reply:
I've removed pyyaml in https://github.com/fedora-infra/bodhi/pull/2897.